app/src/main/java/com/bulletin/MainActivity.kt

```diff
@@ -0,0 +1,147 @@
+package com.example.bulletin
+
+import android.content.Intent
+import android.os.Build
+import android.os.Bundle
+import android.util.Size
+import androidx.appcompat.app.AppCompatActivity
+import androidx.recyclerview.widget.LinearLayoutManager
+import com.bulletin.Appearance
+import com.bulletin.BulletinSdk
+import com.bulletin.FormRecyclerViewAdapter
+import com.bulletin.models.*
+import com.bulletin.utilities.ThemeUtils
+import com.bulletin.utilities.ViewUtil
+import com.example.bulletin.databinding.ActivityMainBinding
+import com.wrx.wazirx.views.bulletin.model.Media
+
+class MainActivity : AppCompatActivity(), FormRecyclerViewAdapter.OnItemClickListener {
+
+    // region Variables
+    private lateinit var binding: ActivityMainBinding
+    private lateinit var formRecyclerViewAdapter: FormRecyclerViewAdapter
+    private var bulletinInfo: ArrayList<BulletinItem> = ArrayList()
+    // endregion
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+
+        super.onCreate(savedInstanceState)
+
+        binding = ActivityMainBinding.inflate(layoutInflater)
+        setContentView(binding.root)
+
+        binding.listView.setBackgroundColor(ThemeUtils.getAttributedColor(R.attr.main_bg_surface_alt, binding.listView.context))
```

Comment:
We are using our colors in this library..?
We should be configuring this from the main app right..?